README.md

```diff
@@ -28,15 +28,14 @@ Live debugger will be running at separate port which you've provided e.g. http:/
 
 ## Adding button
 
-For easy navigation add the debug button to your live layout. Remember to use it only in `:dev` environment if `:live_debugger` is installed as `only: :dev`.
+For easy navigation add the debug button to your live layout. Do not commit it to production! (Since `:live_debugger` is `:dev` only this code won't compile in `:prod` environment)
```

Comment:
```suggestion
For easy navigation add the optional debug button to your live layout. Make sure that it is not used in production! (`:live_debugger` is `:dev` only - this code won't compile in `:prod` environment)
```

Reply:
Maybe we could add approach with config check (not sure if it'll work):
```elixir
    <%= if Application.get_env(:your_app, :include_debug_button) do %>
      <LiveDebugger.Helpers.debug_button socket_id={@socket.id} />
    <% end %>
```

Reply:
Or maybe when LiveDebugger application is started we could set in config sth like this
```elixir
Application.put_env(LiveDebugger, :debug_mode, true)
```

And then next to button 

``` Elixir 
    <%= if Application.get_env(LiveDebugger, :debug_mode, false) do %>
      <LiveDebugger.Helpers.debug_button socket_id={@socket.id} />
    <% end %>
```

Reply:
or we could use `ensure_started/2`
```elixir
    <%= if Application. ensure_started(LiveDebugger) do %>
      <LiveDebugger.Helpers.debug_button socket_id={@socket.id} />
    <% end %>
```

https://hexdocs.pm/elixir/1.12/Application.html#ensure_started/2